.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+<!-- Give some context of this PR. Illustrate with a GIF or a LOOM -->
+
+## What has been done
+
+<!-- how do you solve the problem? -->
+
```

Comment:
## How was it tested

<!-- local environment with ui-library only, integrated within the main project, on which browsers, etc. -->